File: chatwoot/csharp/src/OSEG.ChatwootExamples/UpdateCannedResponseInAccountExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Com.Chatwoot.Api;
using Com.Chatwoot.Client;
using Com.Chatwoot.Model;

namespace OSEG.ChatwootExamples;

public class UpdateCannedResponseInAccountExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey.Add("api_access_token", "USER_API_KEY");

        var cannedResponseCreateUpdatePayload = new CannedResponseCreateUpdatePayload(
        );

        try
        {
            var response = new CannedResponseApi(config).UpdateCannedResponseInAccount(
                accountId: 0,
                id: 0,
                data: cannedResponseCreateUpdatePayload
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling CannedResponseApi#UpdateCannedResponseInAccount: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: chatwoot/typescript-axios/src/UpdateCannedResponseInAccountExample.ts
import axios, { AxiosError } from "axios";
import * as api from "chatwoot_client"

const configuration = new api.Configuration({
  apiKey: "USER_API_KEY",
});

const cannedResponseCreateUpdatePayload: api.CannedResponseCreateUpdatePayload = {
};

new api.CannedResponseApi(configuration).updateCannedResponseInAccount(
  0, // accountId
  0, // id
  cannedResponseCreateUpdatePayload, // data
).then(response => {
  console.log(response.data);
}).catch((error: Error | AxiosError) => {
  console.log("Exception when calling CannedResponseApi#updateCannedResponseInAccount:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
